### PR TITLE
add unknown item type to scheme ls

### PR DESCRIPTION
**Pull request by Valeria1235**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfixsupport to scheme
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

[Issue Number: 74](https://github.com/ydb-platform/ydb-python-sdk/issues/74)

## What is the new behavior?

Support UNKNOWN item type for scheme listing.

### CODE REVIEW

File: ydb/scheme.py [hunk 1]
```diff
@@ -23,6 +23,11 @@ class SchemeEntryType(enum.IntEnum):
     SEQUENCE = 15
     REPLICATION = 16
     TOPIC = 17
+    UNKNOWN = 18
```
Comment (rekby) [inline ydb/scheme.py:26]: It will conflict with protobuf code of next item type.

File: ydb/scheme.py [hunk 2]
```diff
@@ -23,6 +23,11 @@ class SchemeEntryType(enum.IntEnum):
     SEQUENCE = 15
     REPLICATION = 16
     TOPIC = 17
+    UNKNOWN = 1001
```
Comment (rekby) [inline ydb/scheme.py:26]: new code - workaround too, it has universal decisions, which allow grow original codes unlimited and never conflict.

**Review by rekby — CHANGES_REQUESTED** [review]

File: ydb/scheme.py (hunk 2 above)
Comment (Valeria1235) [reply ydb/scheme.py]: fixed it and add some tests

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ## 2.12.0 ##
 * Fixed error message while get token from metadata with asyncio iam
 * Add `SnapshotReadOnly` transaction mode support to `session.transaction`
+* Support `TYPE_UNSPECIFIED` item type to scheme ls
```
Comment (rekby) [inline CHANGELOG.md:4]: changes must be written to top of changelog.

version 2.12.0 is published already and doesn't include TYPE_UNSPECIFIED.

File: ydb/scheme_test.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from ydb.scheme import (
+    SchemeEntryType,
+    _wrap_scheme_entry,
+    _wrap_list_directory_response,
+)
+from ydb._apis import ydb_scheme
+
+
+def test_wrap_scheme_entry():
+    assert (
```
Comment (rekby) [inline ydb/scheme_test.py:10]: Is is necessary to parentheses and new lines inside asserts?

```python
assert _wrap_scheme_entry(ydb_scheme.Entry(type=1)).type is SchemeEntryType.DIRECTORY
```

Comment (gridnevvvit) [reply ydb/scheme_test.py:10]: I guess it's automatic formatting by black library

File: CHANGELOG.md (hunk 3 above)
Comment (Valeria1235) [reply CHANGELOG.md:4]: right, I didn't notice

File: ydb/scheme_test.py (hunk 4 above)
Comment (Valeria1235) [reply ydb/scheme_test.py:10]: yes, this is black's trick

**Review by rekby — APPROVED** [review]

**Pull request merged**